compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -554,12 +554,21 @@ error.invalid.action.invocation=\
 error.undefined.action=\
   undefined remote function ''{0}'' in client object {1}
 
-error.tainted.value.passed.to.sensitive.parameter=\
-  tainted value passed to sensitive parameter ''{0}''
+error.tainted.value.passed.to.untainted.parameter=\
+  tainted value passed to untainted parameter ''{0}''
+
+error.tainted.value.passed.to.untainted.param.in.obj.method=\
+  tainted value ''{0}'' passed to untainted parameter originating from object method invocation
```

Comment:
Can we say which parameter?

Can we say, `tainted value ''{0}'' passed to untainted method parameter ''{1}''`?

Reply:
Here again we can't name the value as it can be result of any expression.
Message changed to `"tainted value passed to untainted parameter ''{0}'' originating from object method ''{1}'' invocation`, which identify name of the untainted parameter being tainted and which method does that.